.licenseignore

```diff
@@ -0,0 +1,12 @@
+**/*.mlmodel
+**/*.pbxproj
+**/*.png
+**/*.strings
+**/*.xcbuildrules
```

Comment:
I don't think we should ignore license headers for strings or xcbuildrules or xcspec files. All of those should (and current do) have a valid license header. The rest make sense.